tidyall.ini

```diff
@@ -4,6 +4,17 @@ argv = --profile=$ROOT/perlcriticrc
 
 [PerlTidy non-auto-generated xt]
 select = xt/author/live/wikipedia.t
+select = lib/**/*.pm
+select = t/**/*.{pm,t}
+select = t/local/log-server
+select = t/local/referer-server
+select = script/*
+ignore = t/00-*
+ignore = t/author-*
+ignore = t/release-*
```

Comment:
Why are these files ignored? It looks like this is called `PerlTidy non-auto-generated xt`, but it looks like it's much more than that with the additions.

Reply:
This is a good point. i don't fully understand what the extra words in the section header mean. I have now added a second `[PerlTidy]` section and grouped the ignores into blocks with comments. 

The `00-*` was too lose. We have one file in the codebase that follows the pattern, and it's fine. One other file is being auto-generated, so that is now ignored explicitly by name. The other we don't have. I believe at the time I copied this from a distribution that used tidyall already.